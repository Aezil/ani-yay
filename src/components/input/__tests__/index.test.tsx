import { render } from '@testing-library/react';
import React from 'react';

import { AniYayInput } from '..';

describe('Ani-Yay Input', () => {
  it('should render Input component with no value', () => {
    const { container } = render(<AniYayInput placeholder='Hello there!' />);
    expect(container).toMatchSnapshot();
  });

  it('should render Input component with a value', () => {
    const { container } = render(<AniYayInput placeholder='Hello there!' value={'GENERAL KENOBI'} />);
    expect(container).toMatchSnapshot();
  });
});
