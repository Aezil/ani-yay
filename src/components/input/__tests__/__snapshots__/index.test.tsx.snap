// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ani-Yay Input should render Input component with a value 1`] = `
<div>
  <input
    class="sc-bczRLJ fPPSOS"
    placeholder="Hello there!"
    value="GENERAL KENOBI"
  />
</div>
`;

exports[`Ani-Yay Input should render Input component with no value 1`] = `
<div>
  <input
    class="sc-bczRLJ fPPSOS"
    placeholder="Hello there!"
  />
</div>
`;
